crates/meilisearch-auth/src/store.rs

```diff
@@ -345,3 +283,35 @@ where
     let head = head.try_into().ok()?;
     Some((head, tail))
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+struct KeyMasks {
+    bitflags: u32, // Bitflags for bit_maks
```

Comment:
Could this be an Action directly?
```suggestion
    bitflags: Action, // Bitflags for bit_maks
```

Reply:
Done